Packs/ArcSightESM/Integrations/ArcSightESMv2/ArcSightESMv2.py

```diff
@@ -848,6 +841,58 @@ def entries_command(func):
     demisto.results("Success")
 
 
+def add_entries_command(args):
+    resource_id = args.get('resourceId')
+    entries_arg = args.get('entries')
+    query_path = f'detect-api/rest/activelists/{resource_id}/entries'
+    fields = []
+    entries = []
+
+    if not isinstance(entries_arg, list):
+        try:
+            entries_arg = json.loads(entries_arg)
+
+        except ValueError as ex:
+            demisto.debug(str(ex))
+            return_error('Entries must be in JSON format. Must be array of objects.')
```

Comment:
Instead of return_error better raise the error again to let main handle the error

Reply:
Same goes for all other `return_error` occurences.